ledger/ledger-api-test-tool-on-canton/canton-dev.conf

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+_shared {
```

Comment:
We could also apply a second conf on top of the first to reduce duplication but if you just want this to be quickly merged, fine for me.